Create an entry that describes the Kui tool within docs (#30598)
kubernetes/website: content/en/docs/reference/tools/_index.md

* Translate a Docker Compose file into Kubernetes objects
* Go from local Docker development to managing your application via Kubernetes
* Convert v1 or v2 Docker Compose `yaml` files or [Distributed Application Bundles](https://docs.docker.com/compose/bundles/)

## Kui

> [`Kui`](https://github.com/kubernetes-sigs/kui) is a tool for enhancing CLIs with Graphics.

Reviewer: ```suggestion
[`Kui`](https://github.com/kubernetes-sigs/kui) is a GUI tool that takes your normal
`kubectl`command line requests and responds with graphics.
```

Kui takes the normal `kubectl` command line requests and responds with graphics. Instead 

> of ASCII tables `Sortable tables` are presented as the output.

Reviewer: ```suggestion
of ASCII tables, Kui provides a GUI rendering with tables that you can sort.
```

> Use Kui to:

Reviewer: ```suggestion
Kui lets you:
```

> * Click the long auto-generated resource names instead of copying and pasting it

Reviewer: ```suggestion
* Directly click on long, auto-generated resource names instead of copying and pasting
```

> * Process `kubectl` commands 2-3 faster than `kubectl` itself

Reviewer: ```suggestion
* Type in `kubectl` commands and see them execute, even sometimes faster than `kubectl` itself
```

> * See waterfall diagrams of your jobs by executing `k get jobs`

Reviewer: ```suggestion
* Query a {{< glossary_tooltip text="Job" term_id="job" >}} and see its execution rendered
  as a waterfall diagram
```

> * Browse through resources in a tabbed UI with help of a click

Reviewer: ```suggestion
* Click through resources in your cluster using a tabbed UI
```
